test/requirements-test.txt

```diff
@@ -1,24 +1,26 @@
+accelerate
```

Comment:
https://github.com/microsoft/Olive/pull/548 Just find a hook to sort the package in requirements-(.*).txt.

Reply:
Thanks! I will rebase this PR if your PR merges before mine and there is conflict. Hopefully there won't be any merge conflicts since the changes look the same.